### DISCUSSION TITLE
how to make path parameter

### DISCUSSION THREAD

**Discussion by ghost** [hoppscotch/hoppscotch#4320] [Q&A] [answered]
is there a way to make path parameter in hoppscotch, i just found query parameter, maybe somethink like this

![image](https://github.com/user-attachments/assets/fe739d51-efa3-4b1c-a1a8-42625f6f25b0)

**Comment by liyasthomas** [comment, marked as answer]
Path variables have a separate tab.

<img width="1133" alt="Screenshot 2024-09-02 at 9 58 47 AM" src="https://github.com/user-attachments/assets/6a2a951a-d1e4-4625-879c-e9ddf871cda5">

<img width="1133" alt="Screenshot 2024-09-02 at 10 04 54 AM" src="https://github.com/user-attachments/assets/e5681a27-14fb-498c-8217-78a7f26fc39d">

**Reply by dangduytung** [reply]
Thanks, good anwser
